common/views/components/IIIFViewerPrototype/ViewerSidebarPrototype.tsx

```diff
@@ -238,7 +238,7 @@ const ViewerSidebarPrototype: FunctionComponent<Props> = ({
             </p>
             {credit && (
               <p>
-                <a href="https:wellcomecollection.org/works/{work.id}">
+                <a href={`https://wellcomecollection.org/works/${work.id}`}>
```

Comment:
Why aren't we using a next link? And also making this a absolute, not relative link?